ocp_resources/data_import_cron.py

```diff
@@ -25,22 +21,26 @@ def __init__(
         managed_data_source=None,
         imports_to_keep=None,
         bind_immediate_annotation=None,
-        teardown=True,
-        privileged_client=None,
-        yaml_file=None,
-        delete_timeout=TIMEOUT_4MINUTES,
         **kwargs,
     ):
-        super().__init__(
-            name=name,
-            namespace=namespace,
-            client=client,
-            teardown=teardown,
-            privileged_client=privileged_client,
-            yaml_file=yaml_file,
-            delete_timeout=delete_timeout,
-            **kwargs,
-        )
+        """
+        Args:
+            garbage_collect (str, optional): whether old PVCs should be cleaned up after a new PVC is imported.
+                Options are "Outdated"/"Never". Default value is "Outdated".
+            imports_to_keep (int, optional): number of import PVCs to keep when garbage collecting. Default value is 3.
+            managed_data_source(str, optional): specifies the name of the corresponding DataSource to manage.
+                DataSource has to be in the same namespace. Default value is "".
```

Comment:
```suggestion
                DataSource has to be in the same namespace.
            
```
The code does not set it to `""`, so should not be included here. pleaser remove in all places